tests/queries/0_stateless/03228_clickhouse_local_copy_argument.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+${CLICKHOUSE_LOCAL} --copy < $CUR_DIR/data_json/twitter.jsonl
```

Comment:
Shellcheck said CUR_DIR does not exist,
there is CURDIR.

Reply:
my bad, fixed